lib/packwerk/application_validator/private.rb

```diff
@@ -0,0 +1,55 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Packwerk
+  class ApplicationValidator
+    module Private
```

Comment:
This is maybe a bit confusing, but hopefully will become less confusing when privacy is totally extracted out. This is just what I typically use to name private constants. That is, I have a module called `Private` and I call `private_constant :Private` at the bottom. I think I can rename this to `Helpers` for now, since I don't want to overload the term "private" too much here.